frontend/src/scenes/sessions/sessionsPlayLogic.ts

```diff
@@ -147,5 +149,44 @@ export const sessionsPlayLogic = kea<sessionsPlayLogicType<SessionPlayerData, Ev
             (index: number, ids: SessionRecordingId[], pagination: Record<string, any> | null) =>
                 index > -1 && (index < ids.length - 1 || pagination !== null),
         ],
+        session: [
+            (selectors) => [selectors.sessionRecordingId, selectors.sessions],
+            (id: SessionRecordingId, sessions: Array<SessionType>): SessionType | null => {
+                const [session] = sessions.filter((s) => s.session_recording_ids.includes(id))
+                return session
+            },
+        ],
+        highlightedSessionEvents: [
+            (selectors) => [selectors.session, selectors.loadedSessionEvents],
+            (session, sessionEvents) => {
+                if (!session) {
+                    return []
+                }
+                const events = session.events || sessionEvents[session.global_session_id] || []
+                return events.filter((e) => (session.action_filter_times || []).includes(e.timestamp))
+            },
+        ],
+        shownPlayerEvents: [
+            (selectors) => [selectors.sessionPlayerData, selectors.eventIndex, selectors.highlightedSessionEvents],
+            (sessionPlayerData, eventIndex, events) => {
+                if (!sessionPlayerData) {
+                    return []
+                }
+                const startTime = +moment(sessionPlayerData.start_time)
```

Comment:
Not sure what the`+` does, but anyways I think related to this might be this case where clicking the label does not take me to the right place in the recording.

![image](https://user-images.githubusercontent.com/5864173/104743954-e8feb800-5711-11eb-8f51-d6e03cac8df5.gif)

Reply:
+ casts the moment object to a number, which can then be used together with the player.

I think what you're seeing is that the two events happen really closely together in time. We don't have a time indicator currently next to these labels - we should.